tfsec.yaml

```
@@ -0,0 +1,48 @@
+package:
+  name: tfsec
+  version: 1.28.5
+  epoch: 0
+  description: Security scanner for your Terraform code
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 2a2a67987b477eb4a76ff42858973096138e9943
+      repository: https://github.com/aquasecurity/tfsec
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: google.golang.org/protobuf@v1.33.0
+
+  - uses: go/build
+    with:
+      packages: ./cmd/tfsec
+      output: tfsec
+      ldflags: -X github.com/aquasecurity/tfsec/version.Version=v${{package.version}} -s -w -extldflags '-fno-PIC -static'
+
+  - uses: go/build
+    with:
+      packages: ./cmd/tfsec-checkgen
+      output: tfsec-checkgen
+      ldflags: -X github.com/aquasecurity/tfsec/version.Version=v${{package.version}} -s -w -extldflags '-fno-PIC -static'
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: aquasecurity/tfsec
+    use-tag: true
+    strip-prefix: v
```

can you please add a basic version test 
```suggestion
    strip-prefix: v
 
 test:
  pipeline:
    - runs: |
        tfsec --version | grep ${{package.version}}
```